docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
> I just don't think any library should read any file.

Literally no `io.Open` calls? That seems both horrifying and gloriously great at the same time to me—but, I think, hardly tenable without a dependency injection framework.

Or do you just mean “should not hardcode any path”? If so, then…

> We can encourage a default like /etc/containers/fetch-policy.d or something through a const package though.

… AFAICS there the following is exactly equivalent in its semantics and capabiliities:

```golang
policy, err = signature.DefaultPolicy(nil)
# and either of
policy, err = signature.NewPolicyFromFile(customPath)
policy, err = signature.DefaultPolicy(&types.SystemContext{SignaturePolicyPath:customPath})
```
(what #49 does)

```golang
policy, err = signature.NewPolicyFromFile(consts.DefaultPolicyPath)
# and
policy, err = signature.NewPolicyFromFile(customPath)
```
(using a const package)

The only difference is that `signature.DefaultPolicy` makes the default case simpler, and `consts.DefaultPolicyPath` makes the non-default case simpler.

> I am also very very hesitant to trust a single file configuration at this point. It places too much burden on tooling to do the right thing. I would rather have systemd style .d directories where you can drop N policies in place.

That’s a very reasonable feature request; can we merge documentation for what the code currently does _first_, and not block that on implementing new features, please?